### DISCUSSION TITLE
How to add a background image to one page?

### DISCUSSION THREAD

**Discussion by banhmr** [mmistakes/minimal-mistakes#2992] [Q&A] [answered]
I'm trying to add a background image to one page (the home page). The header could work temporary but resizing the page will also resize the image.

main.scss
```
body { 
    -webkit-background-size: cover;
    -moz-background-size: cover;
    -o-background-size: cover;
    background-size: cover;
}

body.home {
    background: url(../assets/images/bg.jpg) no-repeat center center fixed; 
}
```

index.html
`<body class="home"></body>`

This is what I currently have but I may be wrong?

**Comment by mmistakes** [comment, marked as answer]
The theme doesn't have a direct way to do what you want other than writing custom CSS and scoping it like you have above.

If you're not doing anything fancy with your CSS and don't need to hook into any of the theme's Sass variables, an easier method would be to add the styles directly to your `index.html` file.

I would just drop them in like this before your content and after any YAML front matter.

```html
---
# your front matter
---

<style>
  body { 
    -webkit-background-size: cover;
    -moz-background-size: cover;
    -o-background-size: cover;
    background-size: cover;
    background: url(../assets/images/bg.jpg) no-repeat center center fixed; 
  }
</style>

Any other content for the home page.
```
